tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/AllTestsSuite.java

```diff
@@ -22,7 +22,7 @@
 
 @RunWith(RedDeerSuite.class)
 @SuiteClasses({
-	ArchetypesTest.class,
+	//ArchetypesTest.class,
```

Comment:
this is safe to delete or whats the reason of commenting this?

Reply:
ArchetypesTest.class contains tests to test the Java EE, Angular JS and HTML 5 projects by accessing them from the Red Hat central. The links to those projects were removed from the Red Hat central (JBIDE-26817) few months ago, so I have decided to comment the test class. To see the changes in Red Hat central (printscreen of Red hat central before and after changes), see the JBIDE-26903 JIRA and the attachment images in that JIRA.

Well, I have a few ideas after all. Maybe I should keep the test class and comment/delete all the 4 tests. After that I can create new tests to test the remaining links in the Red Hat central (OpenShift Application, Launcher Application and Maven project). But maybe it will not be necessary, because those projects are already tested in other tests and they were never tested in ArchetypesTest.class. Maybe we can delete the ArchetypesTest.class completely.

Any suggestion to next steps @jkopriva?

Reply:
then we should delete this class and related files ;)

Reply:
Deleted.